### DISCUSSION TITLE
Loading configuration from SD card is not working

### DISCUSSION THREAD

**Discussion by alexey2020** [BitMaker-hub/NerdMiner_v2#499] [Q&A]
Hi all! I'm using NerdMiner v2 bought from [AliExpress](https://www.aliexpress.com/item/1005007556818632.html).
I'm trying to load configuraiong from the SD card, but nothing happens.
1. I've formatted the card with FA32 system
2. Put there config.json from the example
3. Changed the network name, network password and my bitcoin wallet address
4. I reboot the NerdMiner
Actual Result: nothing happens. It just says "Waiting config" and shows QR-code. When I connect to this network with WiFiManager, everything works fine, so the network settings are OK. I've tried different SD card, but it didn't help. What can be the problem?

**Comment by adisorin** [comment]
I have the same problem. I followed the steps in the instruction and it doesn't load the settings from SD/config.json. Same miner NerdMiner v2.

**Comment by CrocForces** [comment]
6th January 2025. I have formatted microSD as FAT32 as suggested, made config.json file in root, added in structure as given (with my changes). Done reboot on ESP32_2432S028_2USB NerdMiner (v1.7.0 flashed today) and just brings me back to the 'join WiFi' home page. Thanks
